src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -129,14 +129,6 @@ function twentytwenty_theme_support() {
 
 	// Add theme support for selective refresh for widgets.
 	add_theme_support( 'customize-selective-refresh-widgets' );
-
-	/*
-	 * Adds `async` and `defer` support for scripts registered or enqueued
-	 * by the theme.
-	 */
-	$loader = new TwentyTwenty_Script_Loader();
-	add_filter( 'script_loader_tag', array( $loader, 'filter_script_loader_tag' ), 10, 2 );
-
```

Comment:
In the case of child themes that register scripts with `defer` or `async` script data, should we automatically copy these to the `strategy` script data?

Here are the themes that add such script data: https://wpdirectory.net/search/01H9RXP8HXNAD30JZBJY4VRJZ2

However, there is only a single child theme of Twenty Twenty in the directory: https://wpdirectory.net/search/01H9RXTXYNFBGJD9PNY4QJ7CZV

And the `codoc` theme does does not register any of its own scripts.

Reply:
Yeah, I think that's a good idea. By removing this code here, we're not just deprecating but also breaking old usage by child themes, so making sure they still work is crucial. Migrating this data to the script loading strategy sounds great to me.

I am however wondering if deprecating overall is the way to go. What about pre-WP 6.3 sites? We would potentially degrade their performance if we just deprecate this and remove usage. I would prefer we make it an if-else thing based on the WP version and avoid the deprecation of that code (since it's still relevant pre-WP 6.3).

Maybe the cleanest solution would be:
* For any (child theme) script that adds `async` or `defer` script data, migrate to the `strategy` approach regardless of WP version.
* Update the `TwentyTwenty_Script_Loader` class (only relevant for pre-WP 6.3) to rely on `strategy`.

That way we can write the "new" code consistently, and it works across all versions.

Reply:
OK, I've implemented this in 1648b8c3077d2e7b15db4ffae33d195198a14cb5.